### DISCUSSION TITLE
How to get your Namecheap private email to work with Vercel DNS | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#67] [Announcements]
# How to get your Namecheap private email to work with Vercel DNS | Leo's dev blog – stories, insights, and ideas

A step by step guide on how to get your Namecheap private email to work with Vercel DNS.

https://www.leohuynh.dev/blog/how-to-setup-namecheap-private-email-to-work-with-vercel-dns

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺

**Comment by kevinkip** [comment]
This was super helpful. Thank you!

**Reply by hta218** [reply]
Glad it could help @kevinkip 🙌
